integration_tests/helpers/commonHelpers.ts

```diff
@@ -130,3 +130,16 @@ export const deleteS3Events = async (
   console.log("deleted the files from s3");
   return true;
 };
+
+export const regexToRemoveSpecialChar = (str:string): string => {
+  const formatS3Str = str
+    .replace(/:[^"0-9.]*([0-9.]+)/g, ':\\"$1\\"') // converts digits to string for parsing
```

Comment:
had an issue in parsing the txt file to json objects. To overcome the error in parsing converted all digits to string. It would have been easier if we save the textract response and standardised response in json or csv rather than txt file